src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -382,6 +383,22 @@ public void holdCurrentPosition(){
     setBaseReference(getBaseAngle());
     setElbowReference(getElbowAngle());
   }
+
+  public void checkAngles() throws Exception {
+    if(getBaseAngle() > kMaxBaseAngle || 
+    getBaseAngle() < kMinBaseAngle || 
+    getChooChooAngle() > kMaxChooChooAngle ||
+    getChooChooAngle() < kMinChooChooAngle ||
+    getElbowAngle() > kMaxElbowAngle ||
+    getElbowAngle() < kMinElbowAngle) {
```

Comment:
We should make this so it only checks once on enable, and if it doesn't crash it stops checking.

Reply:
done, now stops checking if it passes once on enabled